app/assets/stylesheets/timestamp.scss

```diff
@@ -0,0 +1,138 @@
+.timestamp {
+  background: #eeeeee75;
+  padding: govuk-spacing(3) govuk-spacing(2) govuk-spacing(3) govuk-spacing(4);
+  margin-top: 60px;
+  border-left: govuk-spacing(1) solid #347ca9;
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 14px;
+  font-size: .875rem;
+  line-height: 1.1429;
+
+  @include govuk-media-query($from: tablet) {
+    font-size:16px;
+    font-size: 1rem;
+    line-height: 1.25;
+
+    a {
+      font-family: sans-serif;
+    }
+
+    &--inverse-padded dl {
+      margin: 15px;
+    }
+
+    &__term {
+      box-sizing:border-box;
+      float: left;
+      clear: left;
+      padding-right: 5px;
+      margin-top: 0;
+    }
+
+    &__term dd {
+      line-height: 1.4;
+    }
+
+    &__definition:not(:last-of-type) {
+      margin-bottom: 5px;
+    }
+}
+
+
+
+  a {
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    text-decoration: underline;
+    font-weight: bold;
+    color: govuk-colour('black');
+  }
+
+  &.timestamp::after {
+    content: '';
+    display: block;
+    clear: both;
+  }
+
+  &-inverse {
```

Comment:
Are inverted styles used? I think there are still things in here you can cut away.